ansible/roles/ooni-backend/templates/nginx-api-fsn.conf

```diff
@@ -111,20 +111,20 @@ server {
   # Orchestrate
   # Should match:
   # - /api/v1/test-list
-  location ~^/api/v1/(test-list|urls) {
-      proxy_http_version 1.1;
-      proxy_set_header Host $http_host;
-      proxy_set_header X-Real-IP $remote_addr;
-      proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-      proxy_set_header X-Forwarded-Proto $scheme;
-      proxy_read_timeout 900;
-
-      proxy_pass https://orchestrate.ooni.io:443;
-      add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
-      add_header X-Frame-Options DENY always;
-      add_header X-Content-Type-Options nosniff always;
-
-  }
+  # location ~^/api/v1/(test-list|urls) {
```

Comment:
Maybe let's just add a comment above this to mention we are serving this from ooniapi monolith